File: Python/Vehicle.py
class Vehicle:
    def __init__(self, license_plate, brand, year, color):
        self.__license_plate = license_plate
        self.__brand = brand
        self.__year = year
        self.__color = color

    def get_license_plate(self):
        return self.__license_plate

    def set_license_plate(self, license_plate):
        self.__license_plate = license_plate

    def get_brand(self):
        return self.__brand

    def set_brand(self, brand):
        self.__brand = brand

    def get_year(self):
        return self.__year

    def set_year(self, year):
        self.__year = year

    def get_color(self):
        return self.__color

    def set_color(self, color):
        self.__color = color

File: CPP/Vehicle.cpp
#pragma once

#include <iostream>
#include <string>

using namespace std;

class Vehicle
{
private:
    std::string license_plate;
    std::string brand;
    int year;
    std::string color;

public:
    // Constructor
    Vehicle(std::string license_plate, std::string brand, int year, std::string color)
        : license_plate(license_plate), brand(brand), year(year), color(color) {}

    // Getter and Setter for license plate
    std::string get_license_plate()
    {
        return license_plate;
    }

    void set_license_plate(std::string new_license_plate)
    {
        license_plate = new_license_plate;
    }

    // Getter and Setter for brand
    std::string get_brand()
    {
        return brand;
    }

    void set_brand(std::string new_brand)
    {
        brand = new_brand;
    }

    // Getter and Setter for year
    int get_year()
    {
        return year;
    }

    void set_year(int new_year)
    {
        year = new_year;
    }

    // Getter and Setter for color
    std::string get_color()
    {
        return color;
    }

    void set_color(std::string new_color)
    {
        color = new_color;
    }
};